control/autoware_operation_mode_transition_manager/src/node.cpp

```diff
@@ -210,23 +210,19 @@ void OperationModeTransitionManager::processTransition()
 
 void OperationModeTransitionManager::onTimer()
 {
-  const auto control_mode_report_ptr = sub_control_mode_report_.take_data();
-  if (!control_mode_report_ptr) {
-    return;
-  }
-  const auto gate_operation_mode_ptr = sub_gate_operation_mode_.take_data();
-  if (!gate_operation_mode_ptr) {
+  const auto input_data = subscribeData();
+  if (!input_data) {
```

Comment:
Do not early return here. The `onTimer` function should be executed with gate_operation_mode is UNKNOWN.